jazzhands/__main__.py

```diff
@@ -196,18 +207,24 @@ def main(argv):
 
         if index_files['js'] and js_dir:
             jsx_registry_path = os.path.join(os.path.dirname(index_files['js']), 'jsx_registry.js')
-            subprocess.call(['python', 'manage.py', 'compilejsx'], stdout=open(jsx_registry_path, 'w'))
-            process_jsx(jsx_registry_path)
-            
+            try:
+                import django_jsx
+                subprocess.call(['python', 'manage.py', 'compilejsx'], stdout=open(jsx_registry_path, 'w'))
```

Comment:
Speaking of setup.py - it can make the command script for you during the install so you don't have to code and ship one.  Include something like this:

      entry_points={
        'console_scripts': [
            'jazzhands=jazzhands.__main__:main',
        ]
    }

Reply:
on the first comment: its not actually a dependency of Jazzhands, but it will use it if the _project_ uses it. It imports django_jsx as a way of detecting that it is included in the project (and thus the current virtual environment) and will skip the `compilejsx` command if it is not found.

on the second command: Thanks! Good tip, I'll incorporate that.